exporter/exporter.go

```diff
@@ -348,6 +348,7 @@ func NewRedisExporter(redisURI string, opts Options) (*Exporter, error) {
 		"commands_rejected_calls_total":                {txt: `Total number of errors within command execution per command`, lbls: []string{"cmd"}},
 		"commands_total":                               {txt: `Total number of calls per command`, lbls: []string{"cmd"}},
 		"latency_percentiles_usec":                     {txt: `A summary of latency percentile distribution per command`, lbls: []string{"cmd"}},
+		"latency_usec":                                 {txt: `A histogram of latencies per command`, lbls: []string{"cmd"}},
```

Comment:
Maybe make this `command_latency_usec` to be a bit more descriptive?

Reply:
In that case, would it be better to name it `commands_latencies_usec` instead? This would be more in line with the other metrics prefixed with `commands_`. 

We thus have the alternatives:
1. `redis_command_latency_usec_{bucket,sum,count}`
2. `redis_latency_usec_{bucket,sum,count}`
3. `redis_commands_latencies_{bucket,sum,count}`
4. `redis_commands_latency_{bucket,sum,count}`

Let me know your preference :)

Reply:
`commands_latencies_usec` - I like that